airflow/operators/python.py

```diff
@@ -555,6 +555,13 @@ def __init__(
                 "major versions for PythonVirtualenvOperator. Please use string_args."
                 f"Sys version: {sys.version_info}. Venv version: {python_version}"
             )
+        if type(python_version) is float or type(python_version) is int:
+            warnings.warn(
+                "Passing numeric type(int, float) as python_version is deprecated. "
+                "Please use string value instead.",
+                RemovedInAirflow3Warning,
+                stacklevel=2,
+            )
```

Comment:
I think current version is enough.

should I have to change to this?

If so, I think condition should be not (isinstance(python_version, str) or python_version is None) since if python_version is None, it is interpreted as system's python version.

Reply:
I think current version is enough.

should I have to change to this?

If so, I think condition should be ( not (isinstance(python_version, str) or python_version is None)) since if python_version is None, it is interpreted as system's python version.

Reply:
Oh right, I missed the None case. It is better to check for the allowed cases and raise for anything else instead to avoid weird surprises.

Reply:
Yep. Also using type(object) is generally suspicious. In this case int/float cannot be extended, so no difference, but generally using type rather than isinstance is a code smell.

Reply:
changed statement for readibility. thank you :)